crates/rspack_plugin_css/src/visitors/mod.rs

```diff
@@ -54,7 +59,18 @@ fn replace_module_request_prefix(specifier: String, diagnostics: &mut Vec<Diagno
 }
 
 impl Visit for Analyzer<'_> {
+  fn visit_at_rule(&mut self, n: &AtRule) {
+    if let AtRuleName::Ident(ident) = &n.name && &*ident.value == "import" {
+      self.nearest_at_import_span = Some(n.span);
```

Comment:
This feels weird to me. I think it's better to perform an up-bottom looking up. 

```rust
if let AtRule { prelude: Some(box AtRulePrelude::ImportPrelude(import_predude)), .. } = n {
 // ...
}
```

Reply:
Trying to decouple with `import-prelude` here. For the perspective of `import-prelude`, it should be put into another visit function, which is the same as `support` condition.

Reply:
Hoisting the `import-prelude` function will make this visit function a mess because we have to support `support-condition`, etc in the future.

Reply:
I'm not sure. I don't know how the future goes. But for the current implementation, I'm gonna reserve my opinion on this. An up-bottom looking up is better.

Reply:
Any thoughts on this? @ahabhgk